core/src/themes/test/css-variables/css/default.css

```diff
@@ -9,7 +9,7 @@
  * https://ionicframework.com/docs/theming/color-generator
 **/
 
-:root {
+/* :root {
```

Comment:
can we remove this if we aren't using it anymore?

Reply:
Well it actually was never getting used until my PR which is why I commented it out. The reason I left it in is so that we could see what the defaults are without actually setting them, meaning these variables will go to their fallbacks.